assets/src/scripts/components/hydrograph/index.spec.js

```diff
@@ -147,6 +148,20 @@ const TEST_STATE = {
     },
     ui: {
         width: 400
+    },
+    floodData: {
+        floodLevels: {
```

Comment:
This is not how we are storing the data now, right? We are just storing a single object and not an object with a sites property with an array value?

Reply:
It isn't how state is storing the data, I went ahead and fixed this file as well as three others.